src/main/kotlin/lotto/Rank.kt

```diff
@@ -0,0 +1,20 @@
+package lotto
+
+enum class Rank(val matchCount: Int, val reward: Int) {
+
+    ALLMATCH(6, 2_000_000_000),
```

Comment:
중요한 내용은 아니지만, 아래와 같이 camel로 나누어 질 수 있는 부분에 `_`를 통해 구분해주시면 가독성이 올라갈 것 같아요!

위와 같이 작성하면 IDE의 lint가 초록줄으로 경고를 표시하더라구요.
```suggestion
    ALL_MATCH(6, 2_000_000_000),
```